src/Controls/src/Core/PlatformConfiguration/iOSSpecific/Page.cs

```diff
@@ -161,29 +180,54 @@ public static Thickness SafeAreaInsets(this IPlatformElementConfiguration<iOS, F
 			return config;
 		}
 
-		/// <summary>Bindable property for <see cref="ModalPresentationStyle"/>.</summary>
+		/// <summary>
+		/// Defines the modal presentation style of the Page.
+		/// </summary>
 		public static readonly BindableProperty ModalPresentationStyleProperty =
 			BindableProperty.Create(nameof(ModalPresentationStyle), typeof(UIModalPresentationStyle), typeof(Page), UIModalPresentationStyle.FullScreen);
 
-		/// <include file="../../../../docs/Microsoft.Maui.Controls.PlatformConfiguration.iOSSpecific/Page.xml" path="//Member[@MemberName='ModalPresentationStyle']/Docs/*" />
+		/// <summary>
+		/// Gets the modal presentation style of the Page.
+		/// </summary>
+		/// <param name="config">The platform specific configuration that contains the element on which to perform the operation.</param>
+		/// <returns>The modal presentation style.</returns>
 		public static UIModalPresentationStyle ModalPresentationStyle(this IPlatformElementConfiguration<iOS, FormsElement> config)
 		{
 			return GetModalPresentationStyle(config.Element);
 		}
 
-		/// <include file="../../../../docs/Microsoft.Maui.Controls.PlatformConfiguration.iOSSpecific/Page.xml" path="//Member[@MemberName='SetModalPresentationStyle']/Docs/*" />
+		/// <summary>
+		/// Sets the modal presentation style.
```

Comment:
```suggestion
		/// Sets the modal presentation style of the <see cref="Page"/>.
```

Since you're mentioning Page in the other properties, it might be better to remain consistent.